app/views/admin/extensions/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= form_with(url: toggle_admin_extensions_path, method: :post, local: true) do |f| %>
+  <div class="crayons-card p-4 flex flex-col gap-4">
+    <% @extensions.each do |extension| %>
+      <div>
+        <div class="crayons-field crayons-field--checkbox">
+          <%= f.check_box extension.feature_flag_name, class: "crayons-checkbox", checked: extension.enabled? %>
```

Comment:
```suggestion
          <%= f.check_box extension.feature_flag_name, class: "crayons-checkbox", checked: extension.enabled?, aria: { describedby: "#{extension.feature_flag_name}-description" } %>
```

We should link the description with the checkbox, as this will allow screen reader users to be aware of the relationship between the two / hear the descripton when they visit the input